client/components/ReviewRating/RatingBreakDown/RatingBreakDown.jsx

```diff
@@ -86,15 +86,17 @@ const RatingBreakDown = ({
               <span>
                 {starFilter.map((val) => (
                   <div key={val}>
-                    {String(val)}
-                    {' '}
-                    star
+                    <p>
+                      {String(val)}
+                      {' '}
+                      star
+                    </p>
                     <StarRating size={16} rating={val} />
                   </div>
                 ))}
               </span>
               <div>
-                <button type="button" onClick={clearFilter}>clear</button>
+                <button type="button" onClick={clearFilter} id="clear_btn">clear</button>
```

Comment:
Is there a reason we can't use a class here instead of an id?

Reply:
ok, I use id because I was doing some CSS only for that button. I can change it if ids are for last resort.